style.css

```diff
@@ -1,112 +1,96 @@
-/* stylelint-disable no-missing-end-of-source-newline */
-
-
-/* stylelint-disable no-duplicate-selectors */
-
-
-/* stylelint-disable no-descending-specificity */
+/* stylelint-disable max-empty-lines */
 
 
-/* stylelint-disable declaration-block-no-duplicate-properties */
+/* stylelint-disable no-missing-end-of-source-newline */
 
 
-/* stylelint-disable csstree/validator */
 
+/* stylelint-disable-next-line CssSyntaxError */
 
-/* stylelint-disable color-hex-length */
+/* stylelint-disable rule-empty-line-before */
 
 
 /* stylelint-disable function-calc-no-unspaced-operator */
 
 
-/* stylelint-disable number-no-trailing-zeros */
+/* stylelint-disable declaration-block-no-duplicate-properties */
 
 
-/* stylelint-disable selector-pseudo-element-colon-notation */
+/* stylelint-disable no-descending-specificity */
 
 
 /* stylelint-disable font-family-no-missing-generic-family-keyword */
 
 
-/* stylelint-disable rule-empty-line-before */
-
-
-/* stylelint-disable color-hex-case */
-
-
 /* stylelint-disable indentation */
 
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
 body {
-    font-family: 'Poppins', sans-serif, Arial, Helvetica, 'Lucida Sans', 'Lucida Sans Regular', Lucida, Georgia, 'Times New Roman', Times, serif;
-    width: 100%;
-    height: 100%;
+    font-family: Arial, Helvetica, sans-serif, 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Verdana;
+    justify-content: center;
+    align-items: center;
+    margin: auto;
 }
 
-section {
-    display: none;
+header {
+    width: auto;
+    background: rgba(62, 60, 60, 1);
+    height: 92px;
+    margin: 0;
+    display: flex;
+    align-items: center;
 }
 
-section.active {
-    display: block;
+.hamburger {
+    position: fixed;
+    z-index: 100;
+    top: 1rem;
+    right: 1rem;
+    padding: 4px;
+    cursor: pointer;
 }
 
-#mobile-menu.active {
-    display: block;
+.closeIcon {
+    display: none;
+    padding: 10px;
+    background: white;
+}
+
+.menu {
     position: fixed;
-    transition: transform 59.2s;
-    top: 0;
+    transform: translateY(-100%);
+    transition: transform 0.2s;
     left: 0;
+    top: 0;
     right: 0;
     bottom: 0;
     z-index: 99;
     background: black;
     list-style: none;
+    padding-top: 4rem;
+    margin-top: 0;
 }
 
-.item {
-    display: block;
-    margin: 2rem 4rem;
-    font-size: 1.8rem;
-    color: white;
-    text-decoration: none;
+.menu a {
+    font-size: 39px;
+    color: hsl(25, 100%, 50%);
 }
 
-.item:hover {
-    text-decoration: underline;
-}
-
-main {
-    width: auto;
-    margin: 0;
-    padding: 0;
-}
-
-header {
-    width: 100%;
-    height: 92px;
-    margin-left: auto;
-    margin-right: auto;
+.menu li {
+    margin: 0 5rem 5rem;
+    list-style-type: none;
```

Comment:
- [ ] Please let's ensure that the mobile menu follows closely the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design in terms of  font, colors, images, text, margins.

### Figma Design
![image](https://github.com/richie1988/website-pages/assets/46839548/516023c6-be61-4315-ba42-a8720e3fab1e)


### Your design
![image](https://github.com/richie1988/website-pages/assets/46839548/c68e73ea-1875-4163-9d44-a95e5ee5f954)


In my opinion, you need to do the following to achieve a much similar design 👇 
1. reduce the left margin
2. make sure that the text-color is `#fff5e1`
3. remove the `text-decoration` for the links so that the underline is gone
4. Add a border at the bottom of each link and style it appropriately
5. Ensure that the margin between each mobile menu link is just as in the design
6. Style the close button to look exactly like the Figma design